docs/usage.md

```diff
@@ -131,7 +154,7 @@ samtools bam2fq {prefix}.bam| seqtk seq -a - | gzip - > {prefix}.fasta.gz
 
 <details markdown="1">
   <summary>Details</summary>
-
+Note: This is no longer a necessary step, the pipeline will now do this automatically. I'll keep this here as extra information.
```

Comment:
The GIT history is there, and the version 0.1 is tagged. What about deleting the whole section ?

Reply:
That's a fair point, I've removed it now.